src/connectedk8s/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '1.5.0'
+VERSION = '2.0.0'
```

Comment:
We usually only upgrade the major version when there are some breaking changes in PR. Therefore, it is recommend to upgrade the secondary version for this case

Reply:
This is to compensate for the previous breaking change, which still has a breaking change. I initially upgraded only the minor version, due to some assumptions from the code, but those were wrong. So, I updated the major version to make it clear to customers that there is a breaking change.